packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -802,7 +805,9 @@ class RawGestureDetector extends StatefulWidget {
   /// excluded because the tooltip itself is included in the semantics
   /// tree directly and so having a gesture to show it would result in
   /// duplication of information.
-  final bool excludeFromSemantics;
+  bool get excludeFromSemantics => _semanticsMapping == null;
+
+  final GestureSemanticsMapping _semanticsMapping;
```

Comment:
it's a bit weird that this isn't public, we normally make all the arguments to the constructor map to properties and be public

Reply:
Fixed

Reply:
Fixed by adding a getter for `semanticsMapping`. Or do you think I should store `excludeFromSemantics` as-is?

Reply:
now it's weird because the constructor argument doesn't match the property. usually we do the processing during build rather than construction.

Reply:
Fixed, now both of them are stored directly and an extra getter `_effectiveSemanticsMapping` is used.